keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,418 @@
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and write to it. 
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and read from it. 
+- Ability to schedule multiple containers, with distinct IP addresses, on the same node.
+- Ability to reboot all of the Windows nodes in a cluster while running windows pods, in such a way that recovery of these deleted pods occurs in the scope of a few minutes.
+- Ability to delete and recreate services in such a way that load balancing rules for pods are recovered by whatever chosen service proxy is being utilized.
+- Ability to delete and recreate pods for StatefulSets which preserve their ability to serve as routed endpoints for services, while also having unchanging IP addresses.
+- Ability to access internal pods by internal Kubernetes DNS service endpoints.
+- Ability to access external services by Kubernetes DNS services endpoints (for non-airgapped clusters).
+- Ability to access internal pods through custom DNS records, injected by the Kubernetes pod Specification.
+- Ability to route services from pods from the EndpointSlice API.
+
+#### Basic Service accounts and local storage
+
+- Ability to rotate service accounts for running pods, with the APIServer access capacity of the pod remaining unchanged over the long term.
+- Ability to read and write shared files on a single Kubernetes node between three running Windows containers, simultaneously.
+
+#### Basic Scheduling
+
+- Ability to schedule pods with CPU and Memory limits demonstrably honored over time.
+- Ability to demonstrate the pods requesting more CPU and Memory then available are left in the Pending state over time.
+
+#### Basic performance
+
+- Ability to reschedule the a deployment Pod, 50 times in a row, with continuous deletions in the background.
+- Ability to route traffic to 10 pods behind a common ClusterIP service endpoint.
+- Ability to route traffic to 10 pods behind a common NodePort service endpoint.
+
+We do not include GMSA support in the basic definition of WindowsConformance, because it is known to require other cluster components outside of Kubernete's Control.
+
+### Sub-specifications of Windows Conformance (Kubernetes 1.23)
+
+The following subsets of verifications for Windows clusters further expand the way we define supportability for Windows clusters, but they are not considered part of the "core" Conformance specification.  These features may be *heavily* reliant on functionality which resides well outside of Kubernete's preview, for example: CNI Implementation, Container Runtime implementation, or Windows Server Edition + Configuration.
+
+#### Windows HostProcess Conformance
+
+Note that, HostProcess Containers have not yet been verified to support all of these features.
+Nevertheless, we define this specification because it is an obvious starting point for the long-term, meaningfull support of a full-fledged HostProcess feature in Kubernetes.
```

Comment:
```suggestion
Nevertheless, we define this specification because it is an obvious starting point for the long-term, meaningful support of a full-fledged HostProcess feature in Kubernetes.
```